tests/shared/core/training_data/story_reader/test_yaml_story_reader.py

```diff
@@ -647,27 +575,6 @@ def test_or_statement_story_with_or_slot_was_set(domain: Domain):
     assert training_trackers[1].events[3] == SlotSet(key="name", value="bob")
 
 
-@pytest.mark.parametrize(
-    "file,warning",
-    [
-        ("data/test_yaml_stories/test_base_retrieval_intent_story.yml", None),
-        (
-            "data/test_yaml_stories/non_test_full_retrieval_intent_story.yml",
-            UserWarning,
-        ),
-    ],
-)
-async def test_story_with_retrieval_intent_warns(
```

Comment:
Do we not expect this warning any more?

Reply:
Returned it back — in this case we can actually just remove the optional argument for `YAMLStoryReader` to test exactly the same warning.